nixos/modules/services/monitoring/prometheus/default.nix

```
@@ -851,6 +852,22 @@ in {
       '';
     };
 
+    alertRelabelConfigs = mkOption {
+      type = types.listOf types.attrs;
```

There is actually an existing type for relabling configs:
```suggestion
      type = types.listOf promTypes.relabel_config;
```

---

Good point. Thanks. :-) I changed it.